File: C/3142.c
#include <stdio.h>
#include <math.h>
#include <string.h>

int main(){

    char s[20];

    while(scanf("%s", s) != EOF){
        unsigned long long soma = 0UL;

        for(int i = strlen(s) - 1, j = 0; i >= 0; i--, j++){
            soma += (s[i] - 'A' + 1) * (unsigned long long)pow(26,j);
        }

        if (soma <= 16384){
            printf("%llu\n", soma);
        }else{
            puts("Essa coluna nao existe Tobias!");
        }
    }

    return 0;
}

File: Python/3142.py
import math

while True:
    try:

        s = input()

        n = 0
        j = 0
        for i in range(len(s)-1, -1, -1):
            n += (ord(s[i]) - 64) * math.pow(26, j)
            j += 1

        if n >= 16384:
            print("Essa coluna nao existe Tobias!")
        else:
            print(int(n))

    except EOFError:
        break